scripts/apps/search/components/WidgetItemList.tsx

```diff
@@ -1,86 +1,75 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import {WidgetItem} from './index';
 import {gettext} from 'core/utils';
+import {IArticle} from 'superdesk-api';
+
+interface IProps {
+    allowed?: boolean;
```

Comment:
what is allowed?

Reply:
I didn't change anything about that functionality, I just declared it as a type. We might run into errors if I don't find all the references to it (and it's not on the scope of this task)

Reply:
What do you suggest, to create a JIRA task for renaming a variable? There are only 2 usages of the component, you should be able to fix it in 10 minutes or so.

Reply:
well there are 2 react components (this one and the child) that have that prop. I can change those easily. I meant that value is coming from a `scope.allowed` and that can get messy. But I'll change the components tho

Reply:
My bad, I didn't notice `scope.allowed`. I've just checked the angular part and found that  `WidgetGroup` is getting registered as a directive which is used in only 3 places and doesn't have values coming from another variable with the same name.